pom.xml

```diff
@@ -34,7 +34,7 @@
     <junit.version>4.11</junit.version>
     <commons.io.version>2.6</commons.io.version>
     <commons.lang.version>2.6</commons.lang.version>
-    <hadoop.version>2.8.0</hadoop.version>
+    <hadoop.version>2.3.0</hadoop.version>
```

Comment:
I had to do this change, otherwise the ETL tests initialize fails due to conflicts since cdap core libs depend on hadoop 2.3.

I checked the mvn dependency:tree and found this:
(org.apache.hadoop:hadoop-yarn-api:jar:2.8.0:provided - omitted for conflict with 2.3.0)
and a lot of other similar conflicts between 2.3.0 which pulled by cdap vs 2.8.0 pulled by plugin.

Also I checked all plugins in data-integrations and every plugin which has ETL integration tests is using 2.3.0.